test/e2e/specs/site-editor/page-list.spec.js

```diff
@@ -53,4 +61,356 @@ test.describe( 'Page List', () => {
+						name: 'Choose an image…',
+					} );
+					await placeholder.click();
+					const mediaLibrary = page.getByRole( 'dialog' );
+					const TEST_IMAGE_FILE_PATH = path.resolve(
+						__dirname,
+						'../../assets/10x10_e2e_test_image_z9T8jK.png'
+					);
+
+					const fileChooserPromise =
+						page.waitForEvent( 'filechooser' );
+					await mediaLibrary.getByText( 'Select files' ).click();
+					const fileChooser = await fileChooserPromise;
+					await fileChooser.setFiles( TEST_IMAGE_FILE_PATH );
+					await mediaLibrary
+						.locator( '.media-frame-toolbar' )
+						.waitFor( {
+							state: 'hidden',
+						} );
+
+					await mediaLibrary
+						.getByRole( 'button', { name: 'Select', exact: true } )
+						.click();
+				},
+				assertInitialState: async ( page ) => {
+					const el = page.getByText( 'Choose an image…' );
+					const placeholder = page.getByRole( 'button', {
+						name: 'Choose an image…',
+					} );
+					await expect( el ).toBeVisible();
+					await expect( placeholder ).toBeVisible();
+				},
+				assertEditedState: async ( page ) => {
+					const placeholder = page.getByRole( 'button', {
+						name: 'Choose an image…',
+					} );
+					await expect( placeholder ).toBeHidden();
+					const img = page.locator(
+						'.fields-controls__featured-image-image'
+					);
+					await expect( img ).toBeVisible();
+				},
+			},
+			statusVisibility: {
+				performEdit: async ( page ) => {
+					const statusAndVisibility = page.getByLabel(
+						'Status & Visibility'
+					);
+					await statusAndVisibility.click();
+					const options = [
+						'Published',
+						'Draft',
+						'Pending Review',
+						'Private',
+					];
+
+					for ( const option of options ) {
+						await page
+							.getByRole( 'radio', { name: option } )
+							.click();
+						await expect( statusAndVisibility ).toContainText(
+							option
+						);
+
+						if ( option !== 'Private' ) {
+							await page
+								.getByRole( 'checkbox', {
+									name: 'Password protected',
+								} )
+								.check();
+						}
+					}
+				},
+				assertInitialState: async ( page ) => {
+					const statusAndVisibility = page.getByLabel(
+						'Status & Visibility'
+					);
+					await expect( statusAndVisibility ).toContainText(
+						'Published'
+					);
+				},
+				assertEditedState: async ( page ) => {
+					const statusAndVisibility = page.getByLabel(
+						'Status & Visibility'
+					);
+					await expect( statusAndVisibility ).toContainText(
+						'Private'
+					);
+				},
+			},
+			author: {
+				assertInitialState: async ( page ) => {
+					const author = page.getByLabel( 'Author' );
+					await expect( author ).toContainText( 'admin' );
+				},
+				performEdit: async ( page ) => {
+					const author = page.getByLabel( 'Author' );
+					await author.click();
+					const selectElement = page.locator(
+						'select:has(option[value="1"])'
+					);
+					await selectElement.selectOption( { value: '1' } );
```

Comment:
This is re-selecting the same author (admin), and we don't test the edited state. Perhaps we add a new author as part of the test.